### ISSUE TITLE
[rewrite: test] Add tests for iframe usage

### ISSUE THREAD

**Issue by futurist** [lhorie/mithril.js#1509]
### Description:

Add test for projects using `iframe` with mithril (same domain)

I have a project using this feature, when inspect into `rewrite`, I found it's lack the tests for `iframe`.

### Steps to Reproduce:

Render from **main** into **iframe**

```
var doc = frame.contentDocument || frame.contentWindow.document
m.render(doc.body, m(Component) )
```

Or reversely, render from **iframe** into **main**

```
var doc = window.parent.document
m.render(doc.getElementById('container'), m(Component) )
```

### Expected:

Should add tests for the above situation

### Actual:

Not found tests for the above situation

**Comment by pygy** [comment]
Because of #1390, the rewrite will not handle vdom created in a different realm (i.e. across iframe boundaries).

**Comment by barneycarroll** [comment]
This is now unblocked. What tests do you suggest @futurist?

**Comment by futurist** [comment]
@barneycarroll As far as I can see, the [test-utils](https://github.com/lhorie/mithril.js/tree/rewrite/test-utils) should add `iframe` or `frameset` mocks, or maybe add some real browser cases.

**Comment by isiahmeadows** [comment]
@futurist If you do go that route, you'll need to privately brand the (IIRC) created elements with the proper document so you can do the proper security checks throughout the mocks.

**Comment by isiahmeadows** [comment]
Closing, since this issue has not been updated in over a month, and typically, issues inactive for that long do not usually produce any action. If you feel something in Mithril needs added or changed, please file a new issue.
